test/unit/query/builder.js

```diff
@@ -3606,6 +3606,151 @@ describe("QueryBuilder", function() {
       var result = raw[0].toSQL();
       expect(result.sql).to.equal(raw[1]);
       expect(result.bindings).to.deep.equal(raw[2]);
-    })
-  })
+    });
+  });
+
+  it("wrapped 'with' clause select", function() {
+    testsql(qb().with('withClause', function() {
+      this.select('foo').from('users');
+    }).select('*').from('withClause'), {
+      mssql: 'with [withClause] as (select [foo] from [users]) select * from [withClause]',
+      sqlite3: 'with "withClause" as (select "foo" from "users") select * from "withClause"',
+      postgres: 'with "withClause" as (select "foo" from "users") select * from "withClause"',
```

Comment:
Is it intentional that many of the tests doesn't have oracle / oracledb result checks?